testutils/utils.go

```diff
@@ -21,13 +21,15 @@ import (
 // MakeTempDir creates a directory in the current path for a test. Caller is
 // responsible for managing the uniqueness of the directory name. Returns a
 // function for the caller to delete the temporary directory.
-func MakeTempDir(t *testing.T, tempDir string) func() error {
+func MakeTempDir(t testing.TB, tempDir string) func() error {
 	_, err := os.Stat(tempDir)
 	if !os.IsNotExist(err) {
 		log.Printf("[WARN] temp dir %s was not cleared out after last test. "+
 			"Deleting.", tempDir)
 		err = os.RemoveAll(tempDir)
-		require.NoError(t, err)
+		if err != nil {
```

Comment:
This is fine but I just wanted to point out, in case you were unaware, that `require.NoError()` would still have worked as it takes a stripped down interface that both T and B meet.

Reply:
Fancy! I'll change this back, thanks.